.github/workflows/build-test.yml

```diff
@@ -104,17 +104,17 @@ jobs:
     needs: pre-flight-check
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-    - name: Unix (FreeBSD) test
-      id: Test
-      uses: vmactions/freebsd-vm@v1
-      with:
-        release: "14.0"
-        copyback: false
-        prepare: pkg install -y cmake git ncurses bash wget bison
-        run: |
-         git config --global --add safe.directory /home/runner/work/iceoryx/iceoryx
-         ./tools/ci/build-test-freebsd.sh
+      - uses: actions/checkout@v4
+      - name: Unix (FreeBSD) test
+        id: Test
+        uses: vmactions/freebsd-vm@v1
+        with:
+          release: "14.0"
+          copyback: false
+          prepare: pkg install -y cmake git ncurses bash wget bison
+          run: |
+            git config --global --add safe.directory /home/runner/work/iceoryx/iceoryx
+            ./tools/ci/build-test-freebsd.sh
```

Comment:
Please revert this

Reply:
![image](https://github.com/user-attachments/assets/300c5219-0f0f-46d2-b0e8-72a7df5274e7)
@elBoberido  in the whole file, the formatting is the same as I changed, exactly to bring it back?

Reply:
Oh, in this case thanks for adjusting it :)